### PR TITLE
Get started

### PR DESCRIPTION
**Pull request by RakshithBhat03**
Added Get-Started page for CL.

Deploy Preview: https://deploy-preview-12--clutchui.netlify.app/docs/getstarted

### CODE REVIEW

File: docs/getstarted.html [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        crossorigin="anonymous"></script>
+    <link rel="stylesheet"
+        href="../styles.css" />
+    <title>ClutchUI | Get Started</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <!-- navigation -->
+        <div class="nav">
```
Comment (lannisterrr) [inline docs/getstarted.html]: you can use "header" tag here

**Review by lannisterrr — COMMENTED** [review]
Other than point mentioned above , the code seems fine.

File: docs/getstarted.html (hunk 1 above)
Comment (RakshithBhat03) [reply docs/getstarted.html]: Thanks for pointing it out. Will change it.

File: docs/getstarted.html [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+                                style="
```
Comment (Shivanshsharma01) [inline docs/getstarted.html:144]: little suggestion you can decrease the size of the link by making a small change to code like removing extra space below, background color, padding, or any small property

**Review by Shivanshsharma01 — APPROVED** [review]
The code is clean and well structured. Good to merge. May implement the suggestion

File: docs/getstarted.html (hunk 2 above)
Comment (RakshithBhat03) [reply docs/getstarted.html:144]: Thanks for the suggestion, I will look into it.

**Pull request merged**